rspec_junit_formatter.gemspec

```diff
@@ -12,6 +12,10 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.0.0"
   s.required_rubygems_version = ">= 2.0.0"
 
+  s.metadata = {
+    'changelog_uri' => 'https://github.com/sj26/rspec_junit_formatter/blob/master/CHANGELOG.md',
```

Comment:
Is this a standard?

Reply:
Ah, I see:

https://guides.rubygems.org/specification-reference/#metadata

Reply:
Yup! See the "Metadata" heading at https://guides.rubygems.org/specification-reference/.